src/containers/Review/ReviewPage.tsx

```diff
@@ -44,6 +49,14 @@ const ReviewPage: React.FC<{
 
   if (error) return <Message error title={strings.ERROR_APPLICATION_OVERVIEW} list={[error]} />
   if (!fullReviewStructure) return <Loading />
+
+  // TODO decide how to handle this, and localise if not deleted
+  if (
+    reviewAssignment?.reviewer?.id !== currentUser?.userId &&
+    fullReviewStructure?.thisReview?.status !== ReviewStatus.Submitted
+  )
+    return <Header>Review in Progress</Header>
```

Comment:
Pending... I'm not sure what does this one means. Will find out while testing.

Reply:
Basically when the review is opened, but it's in progress by another reviewer, what do we do ? (there wont' be a link for it, but nothing stopping user accessing it via URL changes

Reply:
I'm still not getting this one. Wouldn't it be set to **Locked**?

Reply:
I've made an issue to discuss this: https://github.com/openmsupply/application-manager-web-app/issues/453